sdk/typescript/src/signers/raw-signer.ts

```diff
@@ -29,14 +28,8 @@ export class RawSigner extends SignerWithProvider {
   }
 
   async signData(data: Uint8Array): Promise<SerializedSignature> {
-    // Starting Sui 0.25.0, only 64-byte nonrecoverable signatures are accepted.
-    // TODO(joyqvq): Remove once 0.25.0 is released.
-    const version = await this.provider.getRpcApiVersion();
-    let useRecoverable =
-      version && gt(versionToString(version), '0.24.0') ? false : true;
```

Comment:
cc @Jordan-Mysten @joyqvq  the usage of femver is actually pretty error-prone. If there were a hotfix commit cherry-picked and deployed to DevNet that caused RPC version to be bumped to `0.24.1`, `useRecoverable` will become `false` unexpectedly. We should almost never use `gt`.